database/index.js

```diff
@@ -0,0 +1,43 @@
+var mysql = require('mysql');
+var mysqlConfig = require('./dbconfig');
+
+var connection = mysql.createConnection(mysqlConfig);
+
+// var getAllUsers = function(res,cb) {
+//   connection.query(`Select * FROM users` , cb)
+// };
+
+const handleResponse = function(err, data, res) {
+  if (err) {
+    res.status(500);
+    res.send(err);
+  } else {
+    res.status(200);
+    res.send(data);
+  }
+}
+
+
+// const getAll = function(res) {
+//   connection.query('SELECT * FROM users', (err, data) => {handleResponse(err, data, res)});
+// }
+
+const getAll = function(res) {
```

Comment:
This function signature should probably include the request and the response objects so that you can use it as middlware.